### PR TITLE
Completed html-forms exercise. Vick Kahlon

### PR DESCRIPTION
**Pull request by vkahlon**
html-form

address bar contains info

### CODE REVIEW

**Comment by vkahlon** [comment]
2nd submission was to make it less confusing for the input type email section

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          <option>Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+
+      <label for="language">Favorite Language (choose one)</label>
+      <div>
+        <input type="radio" name="language" value="HTML" />HTML
```
Comment (codingcodymiller) [inline html-forms/index.html:27]: This text labeling the `input` should be in a `label` element.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        <input type="radio" name="language" value="CSS" />CSS
```
Comment (codingcodymiller) [inline html-forms/index.html:28]: This text labeling the `input` should be in a `label` element.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <input type="radio" name="language" value="JavaScript" />JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:29]: This text labeling the `input` should be in a `label` element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**